packages/core/src/network/network-peers.ts

```diff
@@ -150,26 +201,57 @@ class NetworkPeers {
   public debugLog(): string {
     if (this.peers.size === 0) return 'no connected peers'
 
-    let out = 'current nodes:\n'
+    const peers = this.all()
 
-    // sort peers by quality
-    const peers = Array.from(this.peers.entries()).sort(([, a], [, b]) => {
-      return this.qualityOf(b.id) - this.qualityOf(a.id)
-    })
-    // update map with sorted peers
-    this.peers = new Map(peers)
```

Comment:
if we remove this line, do we lose performance?

Reply:
A map cannot be considered sorted anyway, therefore it has no benefit to overwrite the existing instance with a new one based on a sorted array.

Reply:
The output once you do .values() is certainly changed once given a different input. In any case I wouldn’t block the PR for this, cant speak on how much it improves performance.